src/main/resources/webapp/js/directives.js

```diff
@@ -715,8 +715,12 @@
         var gridDataExtension;
         var rowsToColor = [];
 
-        function createButton(id) {
-            return '<button type="button" class="btn btn-primary btn-sm ng-binding printBtn" ng-click="print()"><i class="fa fa-fw fa-print"></i></button>';
+        function createButton(disabled) {
+            if (disabled) {
```

Comment:
did you try to use ng-disabled instead?

Reply:
Yes I did but it wasn't working correctly. Button was disabled only after clicking on it.